lib/application/README.md

```diff
@@ -0,0 +1,64 @@
+# application
+
+The application module provides an Application class that provides methods to
+queue your users' calls until the actual code to support those calls is
+present.
+
+For example, your user may expect to be able to ask your application to render
+immediately after your scout file is loaded, even though the code to perform
+the rendering has not yet loaded.
+
+Once you create an application instance, it is available via its application
+name as a property in the scout file's namespace. (By default, the scout
```

Comment:
I think it's worth linking to the scoutfile repo somewhere in here if "scout file" in the bv-ui-core docs specifically means that library.